README.md

```diff
@@ -1,6 +1,6 @@
 | Branch | Status                                                                                                                                                                                                                                    | Support level | Programming model | Node.js versions |
 | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- | ----------------- | ---------------- |
-| v2.x   | [![Build Status](https://azfunc.visualstudio.com/Azure%20Functions/_apis/build/status/Azure.azure-functions-durable-js?branchName=v2.x)](https://azfunc.visualstudio.com/Azure%20Functions/_build/latest?definitionId=13&branchName=v2.x) | GA            | V1, V2, V3        | 8.x+             |
+| v2.x   | [![Build Status](https://azfunc.visualstudio.com/Azure%20Functions/_apis/build/status/Azure.azure-functions-durable-js?branchName=v2.x)](https://azfunc.visualstudio.com/Azure%20Functions/_build/latest?definitionId=13&branchName=v2.x) | GA            | V1, V2, V3        | 14.x+            |
```

Comment:
The debate is whether documentation should be historical versions or current versions. I don't really care, but I want us to be consistent (at least within this file). So if you're doing "current" versions, you should change the programming model column to just `V3` and also update line 36 to be core tools "4.x"

Reply:
I don't have a strong opinion on this either. @davidmrdavid thoughts?

Reply:
Let's have this table represent currently supported versions, @hossam-nasr

Reply:
Done. updated to list v3 prog model only and v4 core tools